lib/src/cupertino/cupertino_controls.dart

```diff
@@ -762,7 +763,7 @@ class _CupertinoControlsState extends State<CupertinoControls>
   }
 
   void _startHideTimer() {
-    _hideTimer = Timer(const Duration(seconds: 3), () {
+    _hideTimer = Timer(const Duration(seconds: 2), () {
```

Comment:
can we extract duration as a prop, so it can be customizable?

Reply:
let's keep it for now, I'll need to write fully custom controls anyway when we get the designs